src/branding.rs

```diff
@@ -1,8 +1,44 @@
 #![allow(unused)]
+use const_format::concatcp;
 
-pub const BRANDING: &str = "EdgeDB";
-pub const BRANDING_CLI: &str = "EdgeDB CLI";
-pub const BRANDING_CLOUD: &str = "EdgeDB Cloud";
-pub const BRANDING_CLI_CMD: &str = "edgedb";
+/// The product name.
+pub const BRANDING: &str = if cfg!(feature = "gel") {
+    "Gel"
+} else {
+    "EdgeDB"
+};
+/// The CLI name.
+pub const BRANDING_CLI: &str = concatcp!(BRANDING, " CLI");
+/// The cloud name.
+pub const BRANDING_CLOUD: &str = concatcp!(BRANDING, " Cloud");
+
+/// The CLI command name.
+pub const BRANDING_CLI_CMD: &str = if cfg!(feature = "gel") {
+    "gel"
+} else {
+    "edgedb"
+};
+/// The CLI command name for the alternative executable.
+pub const BRANDING_CLI_CMD_ALT: &str = if cfg!(feature = "gel") {
+    "edgedb"
+} else {
+    "gel"
+};
+/// The executable file name for the CLI.
+pub const BRANDING_CLI_CMD_FILE: &str = if cfg!(windows) {
+    concatcp!(BRANDING_CLI_CMD, ".exe")
+} else {
+    BRANDING_CLI_CMD
+};
+/// The executable file name for the CLI alternative.
+pub const BRANDING_CLI_CMD_ALT_FILE: &str = if cfg!(windows) {
+    concatcp!(BRANDING_CLI_CMD_ALT, ".exe")
+} else {
+    BRANDING_CLI_CMD_ALT
+};
+
+/// The WSL distribution name.
 pub const BRANDING_WSL: &str = "EdgeDB.WSL.1";
+
```

Comment:
Do we want to update this one too?

Reply:
TBH I don't know -- I'm not terribly familiar with WSL. Is there someone who understands what that change might affect? I am not sure if we'd orphan older databases on Windows.